requirements.txt

```diff
@@ -1,121 +1,7 @@
-affine==2.4.0
-aiobotocore==2.13.1
-aiohttp==3.10.2
-aioitertools==0.11.0
-aiosignal==1.3.1
-asciitree==0.3.3
-asli @ git+https://github.com/davidwilby/amundsen-sea-low-index@656914a5fb6c17d4542eedac8f7a622b592e5209
-asttokens==2.4.1
-attrs==23.2.0
-bas-geoplot==0.1.0
-botocore==1.34.131
-branca==0.7.2
-cads-api-client==1.1.0
-Cartopy==0.23.0
-cdsapi==0.7.0
-certifi==2024.7.4
-cftime==1.6.4
-charset-normalizer==3.3.2
-click==8.1.7
-click-plugins==1.1.1
-cligj==0.7.2
-cloudpickle==3.0.0
-comm==0.2.2
-contextily==1.6.0
-contourpy==1.2.1
-cycler==0.12.1
-dask==2024.7.0
-debugpy==1.8.2
-decorator==5.1.1
-executing==2.0.1
-fasteners==0.19
-folium==0.17.0
-fonttools==4.53.1
-frozenlist==1.4.1
-fsspec==2024.6.1
-geographiclib==2.0
-geopandas==1.0.1
-geoplot==0.5.1
-geopy==2.4.1
-idna==3.7
-imageio==2.34.2
-iniconfig==2.0.0
-ipykernel==6.29.5
-ipython==8.26.0
-jedi==0.19.1
-Jinja2==3.1.4
-jmespath==1.0.1
-joblib==1.4.2
-jsonpickle==3.2.2
-jsonschema==4.23.0
-jsonschema-specifications==2023.12.1
-jupyter_client==8.6.2
-jupyter_core==5.7.2
-kiwisolver==1.4.5
-lazy_loader==0.4
-locket==1.0.0
-mapclassify==2.7.0
-MarkupSafe==2.1.5
-matplotlib==3.9.1
-matplotlib-inline==0.1.7
-mercantile==1.2.1
-meshiphi==2.1.11
-multidict==6.0.5
-multiurl==0.3.1
-nest-asyncio==1.6.0
-netCDF4==1.7.1.post1
-networkx==3.3
-numcodecs==0.13.0
-numpy==2.0.0
-packaging==24.1
-pandas==2.2.2
-parso==0.8.4
-partd==1.4.2
-pexpect==4.9.0
-pillow==10.4.0
-platformdirs==4.2.2
-pluggy==1.5.0
-prompt_toolkit==3.0.47
-psutil==6.0.0
-ptyprocess==0.7.0
-pure-eval==0.2.2
-Pygments==2.18.0
-pyogrio==0.9.0
-pyparsing==3.1.2
-pyproj==3.6.1
-pyshp==2.3.1
-pytest==8.3.2
-python-dateutil==2.9.0.post0
-pytz==2024.1
-PyYAML==6.0.1
-pyzmq==26.0.3
-rasterio==1.3.10
-referencing==0.35.1
-requests==2.32.3
-rioxarray==0.17.0
-rpds-py==0.19.1
+asli[tests,docs,dev,s3] @ git+https://github.com/davidwilby/amundsen-sea-low-index
```

Comment:
This looks great, much more contained.

Does the asli installation need the docs optional deps here?

Optionally, you could also free up the package versions using `>=` or something, but pinning to exact versions is fine too.